format-registry/initdata/index-respec.html

```diff
@@ -153,19 +153,19 @@
             <td>
               cenc
             </td>
-            <td><a href="cenc.html">ISO Common Encryption Protection System Specific Header (PSSH) Boxes Initialization Data Format</a></td>
+            <td>[[!EME-INITDATA-CENC]]</td>
```

Comment:
This is inconsistent with the change to format-registry/stream/index-respec.html. Such indirection here is a huge hit to usability of the landing page.